extensions/amp-story/1.0/amp-story-open-page-attachment.js

```diff
@@ -38,13 +40,49 @@ export const buildOpenAttachmentElement = (element) =>
     </a>`;
 
 /**
- * Renders the open attachment UI affordance.
+ * @param {!Element} element
+ * @return {!Element}
+ */
+export const buildOpenInlineAttachmentElement = (element) =>
+  htmlFor(element)`
+    <a class="
+        i-amphtml-story-page-open-attachment i-amphtml-story-system-reset"
+        role="button" aria-label="Swipe up">
```

Comment:
Please remove the untranslated aria-label, it shouldn't be used, we add it programmatically using the i18n service.

Reply:
Is there ever a case were the service wouldn't load and this would be helpful as a fallback?

Reply:
I don't think so, but tbh I'm mostly worried about duplicating the strings and/or forgetting to i18n like we did in this PR.